### PR TITLE
fix: docs link should open in a new tab

### PR DESCRIPTION
**Pull request by Manav2808**
Closes: [Info message link in bridge opens in the same browser tab #251](https://github.com/FuelLabs/fuel-explorer/issues/251)

### CODE REVIEW

File: packages/app-portal/src/systems/Bridge/pages/Bridge.tsx [hunk 1]
```diff
@@ -122,12 +121,13 @@ export const Bridge = () => {
           Any assets deposited to Fuel can take up to {timeToWithdrawFormatted}{' '}
           to withdraw back to Ethereum. Learn more about our architecture and
           security in our&nbsp;
-          <Link
-            isExternal
+          <a
```
Comment (helciofranco) [inline packages/app-portal/src/systems/Bridge/pages/Bridge.tsx:124]: We just need change to keep using that `Link`.
It adds a pretty icon and other styles to it.

You can keep `target` and `rel` as you did. That looks good.

```suggestion
          <Link
```

**Review by helciofranco — CHANGES_REQUESTED** [review]
Hey there! 

Thanks for you contribution, left just a small comment.

File: packages/app-portal/src/systems/Bridge/pages/Bridge.tsx (hunk 1 above)
Comment (Manav2808) [reply packages/app-portal/src/systems/Bridge/pages/Bridge.tsx]: Done.

File: packages/app-portal/src/systems/Bridge/pages/Bridge.tsx [hunk 2]
```diff
@@ -123,8 +123,9 @@ export const Bridge = () => {
           to withdraw back to Ethereum. Learn more about our architecture and
           security in our&nbsp;
           <Link
-            isExternal
+            target="_blank"
```
Comment (helciofranco) [inline packages/app-portal/src/systems/Bridge/pages/Bridge.tsx:126]: You missed the `isExternal` prop (that property is adding an icon there)

```suggestion
            isExternal
            target="_blank"
```

**Review by helciofranco — CHANGES_REQUESTED** [review]

File: packages/app-portal/src/systems/Bridge/pages/Bridge.tsx (hunk 2 above)
Comment (Manav2808) [reply packages/app-portal/src/systems/Bridge/pages/Bridge.tsx:126]: Oh, my bad. Now Done.

**Pull request merged**